client/components/editor/Sidebar/Navigation.vue

```diff
@@ -11,13 +11,23 @@
         class="my-3 mx-1" />
     </v-sheet>
     <v-treeview
-      @update:active="navigateTo"
+      ref="activityTree"
       :items="activityTree"
-      :active.sync="active"
+      :active="active"
       :search="search"
-      open-all activatable hoverable dense>
-      <template v-slot:label="{ item: { name, selectable } }">
-        <span :class="{ selectable }">{{ name }}</span>
+      open-all
+      hoverable>
+      <template v-slot:label="{ item: { id, name, selectable } }">
+        <div @click.stop="navigateTo(id)" :class="{ selectable }">
+          {{ name }}
+          <v-btn
```

Comment:
It was a button because it was the only one clickable. Not the wrapper is. So I will remove that button but the `div` should be changed to a `button` since it is clickable. Do you agree?